docs/api/features/placement.md

```diff
@@ -1,3 +1,137 @@
 # Placement
 
-What are the default rules in SDC, do they differ in hosted Triton?
+When using Docker with a Triton data center (a.k.a. SDC) as your Docker Host,
+your Docker containers can be placed on any physical server in the data center.
+This is one of the basic benefits of Triton's Docker solution: with just the
+vanilla `docker` client you can get the benefit of spreading your containers
+across multiple servers, setting the groundwork for high availability (HA)
+services and avoiding single points of failure (SPOF).
+
+This document explains how Triton places new containers on servers and what
+facilities are exposed for controlling placement.
+
+
+## Default placement
+
+By default, Triton attempts to spread all containers (and non-Docker containers
+and VMs) owned by a single account across separate physical servers.
+
+Within a Docker container the physical server on which a container is running
+is exposed via the [`sdc:server_uuid` metadata
+key](http://eng.joyent.com/mdata/datadict.html):
+
+    $ docker run -ti --rm alpine /bin/sh
+    / # hostname
+    cc72c36cb591
+    / # /native/usr/sbin/mdata-get sdc:server_uuid
+    44454c4c-5400-1034-8052-b5c04f383432
+
+By running another container we can see this default spread behavior (the
+`server_uuid` differs):
+
+    $ docker run -ti --rm alpine /bin/sh
+    / # hostname
+    98d5a22a0c5e
+    / # /native/usr/sbin/mdata-get sdc:server_uuid
+    44454c4c-4400-1054-8052-b5c04f383432
+
+Outside the containers, the physical server on which a container is running
+is exposed via the `compute_node` field using the [Triton CLI](XXX):
+
+    $ triton insts -o shortid,name,age,compute_node
+    SHORTID   NAME             AGE  COMPUTE_NODE
+    cc72c36c  serene_bose      6m   44454c4c-5400-1034-8052-b5c04f383432
+    98d5a22a  goofy_engelbart  2m   44454c4c-4400-1054-8052-b5c04f383432
+
+
+## Swarm affinity
+
+[Docker Swarm](https://docs.docker.com/swarm/overview/) is a system to allow you
+to talk to a pool of Docker hosts (a.k.a. nodes) as a single virtual Docker
+host. `docker run` against a Swarm master will choose a node on which to run
+your container. Sometimes it matters which node is selected -- my
+webhead should run near the redis is uses for caching, this database instance
```

Comment:
@misterbisson Typo. "near the redis it uses for caching" ... is that clear then?